coveo-testing/README.md

```diff
@@ -0,0 +1,18 @@
+# coveo-testing
+
+A set of test/pytest helpers to facilitate common routines.
+
+
+Content in a nutshell:
+
+- Reusable pytest markers (UnitTest, IntegrationTest)
+- TemporaryResource fixture class with unique ID generation
+- Multiline logging assertions with includes, excludes, levels and comprehensive assertion output
+- Refactorable "unittest.mock.patch('this.module')" module references
+- Human-readable (but still customizable) display for parametrized tests

```

Comment:
Again, I think the documentation would significantly benefit from an example by bullet point.

Reply:
[x] - examples for coveo-testing